scripts/systemd/falcoctl-artifact-follow.service

```diff
@@ -13,7 +13,6 @@ RestartSec=15s
 Restart=on-failure
 PrivateTmp=true
 NoNewPrivileges=yes
-ProtectSystem=true
```

Comment:
Can we try `ReadWritePaths` instead?

Reply:
It looks like `ReadWritePaths` was introduced in systemd 231: https://github.com/systemd/systemd/blob/main/NEWS#L9059

I think we should use `ReadWriteDirectories` that is kept for backward compatibility:
> The old names of these settings remain
          available for compatibility.

Reply:
In falco-kmod.service too!